test/lib/output/FakeResults.ts

```diff
@@ -0,0 +1,60 @@
+import {Results} from "../../../src/lib/output/Results";
+import {EngineExecutionSummary, FormattedOutput, RuleResult} from "../../../src/types";
+import {OutputFormat} from "../../../src/lib/output/OutputFormat";
+
+export class FakeResults implements Results {
+	private minSev: number = 0;
+	private summaryMap: Map<string, EngineExecutionSummary>;
+	private formattedOutputMap: Map<string, FormattedOutput> = new Map();
+
+	withMinSev(minSev: number): FakeResults {
+		this.minSev = minSev;
+		return this;
+	}
+
+	withSummaryMap(summaryMap: Map<string, EngineExecutionSummary>): FakeResults {
+		this.summaryMap = summaryMap;
+		return this;
+	}
+
+	withFormattedOutput(formattedOutput: FormattedOutput): FakeResults {
+		this.formattedOutputMap.set("default", formattedOutput);
+		return this;
+	}
+
+	withFormattedOutputForFormat(format: OutputFormat, formattedOutput: FormattedOutput): FakeResults {
```

Comment:
I mostly just moved this class to its own file to share it between our tests. Note that the only change was to switch to a formattedOutputMap so that we can specify each formatted output based on the format.